src/app/app.html

```diff
@@ -9,6 +9,10 @@
     <ion-content>
       <ion-list role="navigation" id="navigation-menu" aria-label="Peeveetrack Menu">
         <button role="link" menuClose ion-item *ngFor="let p of pages" (click)="openPage(p)">
+        <ion-icon name="ios-heart-outline" *ngIf="p.title == 'Favorites'"></ion-icon>
```

Comment:
Do you mean writing a name constant and changing it according to the page title? Or to throw this in a different place? Like in the typescript file or something?